Solutions/AzureDevOpsAuditing/Hunting Queries/ADOInternalUpstreamPacakgeFeedAdded.yaml

```diff
@@ -1,6 +1,8 @@
 id: 20be967c-4923-4c4b-8e1d-e1c95d537dc3
 name: Azure DevOps - Internal Upstream Package Feed Added
 description: |
+  'This query monitors for compromised packages in builds, advising initial use as a hunting query due to false positives, with potential upgrade to detection in low-event environments.'
+description_detailed: |
   'An attacker aiming to insert malicious code into a build process could look to introduce compromised upstream packages into the build process. Looking at internal packages can have a significant false positive rate compared to looking at external feeds so running this as a hunting query at least initially is advised. If an environment has low number of events it can be upgraded to a detection.'
```

Comment:
Attackers may introduce upstream packages into the build process to insert malicious code. This query searches for such malicious activity. If an environment has low number of events, it can be upgraded to a detection.